src/ssl.c

```diff
@@ -6649,7 +6649,7 @@ int ProcessFile(WOLFSSL_CTX* ctx, const char* fname, int format, int type,
         dynamic = 1;
     }
 
-    if ( (ret = (int)XFREAD(myBuffer, 1, sz, file)) != sz)
+    if ((long int)XFREAD(myBuffer, 1, sz, file) != (long int)sz)
```

Comment:
Please use `size_t` not `long int`. Same below

Reply:
It would be `ssize_t`, not `size_t`, and the only reason I didn't use it was concern about platform support for `ssize_t`.  `ssize_t` is currently used only in `./wolfssl/test.h` and, umm, `wc_XChaCha20Poly1305_crypt_oneshot()`, where I used it thoughtlessly.

What do you think?  Use `ssize_t` here, or get rid of `ssize_t` in `wc_XChaCha20Poly1305_crypt_oneshot()`?

Reply:
Whoops, I misread `XFREAD()` as `XREAD()`.  As it happens, there _is_ no `XREAD()` macro in the codebase, but there are numerous instances of `read()` being used as such, and the retvals are consistently treated with signed math and types.  Nothing to see here; changing these to `size_t` as you suggest.  "We regret the inconvenience."